### PR TITLE
New name & first exercise

### PR DESCRIPTION
**Pull request by Felienne**

### CODE REVIEW

**Review by katjam — APPROVED** [review]
This looks great. You've done so much work to get this ready!

Let's go ahead and merge. I'll promise to get my code snippets ready this week. I have a couple ideas, I just need to write it up.

File: content/blog/countdown-3.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Today's code reading countdown.
+
+**Snippet**
+
+![CRCRC-3](/images/articles/CRCRC-3.png)
+
+**Question**
+
+Which JavaScript behavior(s) and feature(s) does t his code exploit to *not* break where other languages might protest?
```
Comment (Nikomus) [inline content/blog/countdown-3.md:25]:
```suggestion
Which JavaScript behavior(s) and feature(s) does this code exploit to *not* break where other languages might protest?
```

**Review by Nikomus — COMMENTED** [review]
Are we able to add a ``` formatted version of the code into these pages? It would be good for accessibility to have a text representation - but otherwise looks great to me!

**Comment by Felienne** [comment]
> Are we able to add a ``` formatted version of the code into these pages? It would be good for accessibility to have a text representation - but otherwise looks great to me!

That is a good idea! Let me add that.

**Pull request merged**